account_invoice_report_commercial_address/readme/CONFIGURE.md

```diff
@@ -0,0 +1,7 @@
+Installation
```

Comment:
Installation instructions, in cases when they're needed, must be defined in its own file